app/javascript/admin/adminModal.js

```diff
@@ -4,28 +4,32 @@
  * @function adminModal
  * @param {string} title The title of the modal.
  * @param {string} body The modal's content. May use HTML tags for styling.
- * @param {string} confirmBtnText The text for the modal's "Confirm" button.
- * @param {string} confirmBtnAction The function that fires when "Confirm" button is clicked.
- * @param {string} cancelBtnText The text for the modal's "Cancel" button.
- * @param {string} cancelBtnAction The function that fires when "Cancel" button is clicked.
+ * @param {string} firstBtnText The text for the modal's first button.
+ * @param {string} firstBtnAction The function that fires when first button is clicked.
+ * @param {string} secondBtnText The text for the modal's second button.
+ * @param {string} secondBtnAction The function that fires when second button is clicked.
+ * @param {string} firstBtnClasses Classes applied to first button.
+ * @param {string} secondBtnClasses Classes applied to second button.
  * @param {string} customAttr A custom data attribute name. Will be apprended to the "data-" part.
  * @param {string} customAttrValue The value of the custom attribute "customAttr".
  */
 const adminModal = (
   title,
   body,
-  confirmBtnText,
-  confirmBtnAction,
-  cancelBtnText,
-  cancelBtnAction,
+  firstBtnText,
+  firstBtnAction,
+  secondBtnText,
+  secondBtnAction,
+  firstBtnClasses,
+  secondBtnClasses,
```

Comment:
It might be more descriptive to call these `leftButton` and `rightButton` to help make the rendering/usage of these buttons a bit more clear. To me, `firstBtn` isn't as clear as `confirmButton`, but I totally get why you switched the naming here. I think that `leftButton` is a bit clearer because, as a developer, I know where this will _render_; `firstBtn` doesn't necessarily make that as clear (and the documentation we have for it, `The text for the modal's first button.`, also doesn't explain that; a developer would have to look at the markup directly to figure out what the "first button" does)

Reply:
I agree :)

Reply:
question: Are the button names more generic because for example, the left button isn't always a confirm button? I know it's been changed, but `confirmButton` provides more context than `leftButton`.

Reply:
Yes @nickytonline you are right. I made this modal as generic as possible so it can be repurposed as needed.